pydantic_ai_slim/pydantic_ai/mcp.py

```diff
@@ -303,16 +338,33 @@ async def main():
     If `None`, no log level will be set.
     """
 
+    tool_prefix: str | None = None
+    """A prefix to add to all tools that are registered with the server.
+
+    If not empty, will include a trailing underscore(`_`).
+
+    e.g. if `tool_prefix='foo'`, then a tool named `bar` will be registered as `foo_bar`
```

Comment:
Can you apply this in the other docstrings as well?

```suggestion
    If not empty, will include a trailing underscore (`_`).

    For example, if `tool_prefix='foo'`, then a tool named `bar` will be registered as `foo_bar`
```